internal/app/feed/feed_user.go

```diff
@@ -11,9 +11,9 @@ import (
 
 // User represents a row from the users feed
 type User struct {
-	ID             string    `json:"id" csv:"user_id" gorm:"primarykey;column:user_id"`
-	OrganisationID string    `json:"organisation_id" csv:"organisation_id"`
-	Email          string    `json:"email" csv:"email"`
+	ID             string    `json:"id" csv:"user_id" gorm:"primarykey;column:user_id;size:37"`
+	OrganisationID string    `json:"organisation_id" csv:"organisation_id" gorm:"size:37"`
+	Email          string    `json:"email" csv:"email" gorm:"size:256"`
 	Firstname      string    `json:"firstname" csv:"firstname"`
```

Comment:
We potentially have platform limits on this that you could enforce.

Reply:
The reason we adding fixed sizes is to allow customers add indexes to those columns. I don't think anyone going to create indexes on these properties.